cli/sawtooth_cli/cluster.py

```diff
@@ -220,7 +232,12 @@ def do_cluster_start(args):
     state["DesiredState"] = "Running"
 
     if args.processors is None:
-        raise CliException("Use -P to specify one or more processors")
+
+        error_string = '''Use -P to specify one or more processors.\n
+        Enter command "sawtooth cluster start -h" for more details."'''
```

Comment:
Remove the newline and 2nd line.  It is not common for Linux commands to point out -h or --help in error messages as they are very standard options.